### PR TITLE
Fix typo in persistence docs

### PR DESCRIPTION
**Pull request by sbrunk**

### CODE REVIEW

File: akka-docs/rst/java/persistence-schema-evolution.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ Picking the right serialization format
 
 Picking the serialization format is a very important decision you will have to make while building your application.
 It affects which kind of evolutions are simple (or hard) to do, how much work is required to add a new datatype, and,
-last but not least, serialization performanceion.
+last but not least, serialization performance.
```
Comment (patriknw) [inline akka-docs/rst/java/persistence-schema-evolution.rst]: :-)

**Comment by patriknw** [comment]
thanks

**Comment by patriknw** [comment]
why is not CLA validator running here? DON'T MERGE UNTIL THAT IS DONE

**Comment by drewhk** [comment]
He has signed the CLA, I checked manually

**Comment by patriknw** [comment]
thanks for contributing @sbrunk

**Pull request merged**